### PR TITLE
Addresses #2000 NTR retinal ganglion cells

### LINKED ISSUE #2000: [NTR] Retinal ganglion cells
**Preferred term label**
ON midget ganglion cell (PMID:12675488)

**Synonyms**
ON-midget RGC (PMID:34093409), ON-midget cell (PMID:34093409), inner stratifying midget cell (PMID: 32032773)

**Definition** 
A midget ganglion cell that depolarises by increased illumination of their receptive field center. The majority of input that this cell receives provides from invaginating midget bipolar cells. (PMID: 32032773)

**Parent cell type term** 
midget ganglion cell of retina (CL:4023188)
ON retinal ganglion cell (CL:4023032)

**What is the anatomical structure that the cell is a part of?**
'has soma location' some ganglionic layer of retina (UBERON:0001792)

------

**Preferred term label**
OFF midget ganglion cell (PMID:12675488)

**Synonyms**
OFF-midget RGC (PMID:34093409), OFF-midget cell (PMID:34093409), outer stratifying midget cell (PMID: 32032773)

**Definition** 
A midget ganglion cell that depolarises by decreased illumination of their receptive field center. The majority of input that this cell receives provides from flat midget bipolar cells. (PMID: 32032773)

**Parent cell type term** 
midget ganglion cell of retina (CL:4023188)
OFF retinal ganglion cell (CL:4023033)

**What is the anatomical structure that the cell is a part of?**
'has soma location' some ganglionic layer of retina (UBERON:0001792)

------

**Preferred term label**
ON parasol ganglion cell

**Synonyms**
ON-parasol RGC (PMID:34093409), ON-parasol retinal ganglion cell (PMID:34093409), inner parasol cell (PMID: 32032773)

**Definition** 
A parasol ganglion cell that depolarises by increased illumination of their receptive field center. The majority of input that this cell receives provides from DB4 bipolar cells. (PMID: 32032773)

**Parent cell type term** 
parasol ganglion cell of retina (CL:4023189)
ON retinal ganglion cell (CL:4023032)

**What is the anatomical structure that the cell is a part of?**
'has soma location' some ganglionic layer of retina (UBERON:0001792)

------

**Preferred term label**
OFF parasol ganglion cell

**Synonyms**
ON-parasol RGC (PMID:34093409), ON-parasol retinal ganglion cell (PMID:34093409), inner parasol cell (PMID: 32032773)

**Definition** 
A parasol ganglion cell that depolarises by decreased illumination of their receptive field center. The majority of input that this cell receives provides from DB3a bipolar cells. (PMID: 32032773)

**Parent cell type term** 
parasol ganglion cell of retina (CL:4023189)
OFF retinal ganglion cell (CL:4023033)

**What is the anatomical structure that the cell is a part of?**
'has soma location' some ganglionic layer of retina (UBERON:0001792)

------

**Preferred term label**
small bistratified retinal ganglion cell

**Synonyms**
sbRGC PMID:34093409, small bistratified cell, blue-ON/yellow-OFF ganglion cell ( https://doi.org/10.1002/cne.22183), SBS ganglion cell (PMID:34093409)

**Definition** 
A retinal ganglion cell that has dendrites in the ON and OFF sublamina of the retinal inner plexiform layer and carries blue-ON/yellow-OFF signals. This cell receives bipolar and amacrine input to both the OFF and ON dendritic tree. (PMID: 32032773)

**Parent cell type term** 
bistratified retinal ganglion cell (CL:0003001)

**What is the anatomical structure that the cell is a part of?**
'has soma location' some ganglionic layer of retina (UBERON:0001792)

**Comment**

### PR DESCRIPTION
**Pull request by aleixpuigb**
Fixes #2000 NTR retinal ganglion cells

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -31771,6 +31798,42 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10191060") Annotation(ob
 AnnotationAssertion(rdfs:label obo:CL_4033050 "catecholaminergic neuron")
 EquivalentClasses(obo:CL_4033050 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0160043)))
 
+# Class: obo:CL_4033051 (OFF parasol ganglion cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") obo:IAO_0000115 obo:CL_4033051 "A parasol ganglion cell that depolarizes in response to decreased light intensity in the center of its receptive field. The majority of input that this cell receives provides from DB3a bipolar cells.")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31803]: "provides" --> "comes"

File: src/ontology/cl-edit.owl [hunk 2]
```diff
@@ -31771,6 +31798,42 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10191060") Annotation(ob
...
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033051 "2023-08-08T12:34:57Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34093409") oboInOwl:hasExactSynonym obo:CL_4033051 "OFF-parasol RGC")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34093409") oboInOwl:hasExactSynonym obo:CL_4033051 "OFF-parasol retinal ganglion cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") oboInOwl:hasExactSynonym obo:CL_4033051 "outer parasol cell")
+AnnotationAssertion(rdfs:label obo:CL_4033051 "OFF parasol ganglion cell")
+EquivalentClasses(obo:CL_4033051 ObjectIntersectionOf(obo:CL_4023033 obo:CL_4023189))
+
+# Class: obo:CL_4033052 (ON parasol ganglion cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") obo:IAO_0000115 obo:CL_4033052 "A parasol ganglion cell that depolarizes in response to increased light intensity in the center of its receptive field. The majority of input that this cell receives provides from DB4 bipolar cells.")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31814]: "provides" --> "comes"

File: src/ontology/cl-edit.owl [hunk 3]
```diff
@@ -31771,6 +31798,42 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10191060") Annotation(ob
...
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033053 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033053 "2023-08-08T12:43:56Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") oboInOwl:hasExactSynonym obo:CL_4033053 "small bistratified cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1002/cne.22183") oboInOwl:hasExactSynonym obo:CL_4033053 "small bistratified cell, blue-ON/yellow-OFF ganglion cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") oboInOwl:hasRelatedSynonym obo:CL_4033053 "Polyak's shrub cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34093409") oboInOwl:hasRelatedSynonym obo:CL_4033053 "SBS ganglion cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34093409") oboInOwl:hasRelatedSynonym obo:CL_4033053 "sbRGC")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31832]: synonym type = abbreviation

File: src/ontology/cl-edit.owl [hunk 4]
```diff
@@ -31771,6 +31798,42 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10191060") Annotation(ob
...
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") obo:IAO_0000115 obo:CL_4033053 "A bistratfied retinal ganglion cell with a small dendritic field that has dendrites in the ON and OFF sublamina of the retinal inner plexiform layer and carries blue-ON/yellow-OFF signals. This cell receives bipolar and amacrine input to both the OFF and ON dendritic tree.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033053 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033053 "2023-08-08T12:43:56Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") oboInOwl:hasExactSynonym obo:CL_4033053 "small bistratified cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1002/cne.22183") oboInOwl:hasExactSynonym obo:CL_4033053 "small bistratified cell, blue-ON/yellow-OFF ganglion cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") oboInOwl:hasRelatedSynonym obo:CL_4033053 "Polyak's shrub cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34093409") oboInOwl:hasRelatedSynonym obo:CL_4033053 "SBS ganglion cell")
```
Comment (bvarner-ebi) [inline src/ontology/cl-edit.owl:31831]: SBS found in PMID:32032773, not PMID:34093409.

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
Please see inline comments.

**Comment by aleixpuigb** [comment]
Thank you for your comments, @bvarner-ebi. I have applied your suggestions.

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**